test/ae_mdw_web/controllers/stats_controller_test.exs

```diff
@@ -229,4 +229,123 @@ defmodule AeMdwWeb.StatsControllerTest do
              |> get(prev_url)
              |> json_response(200)
   end
+
+  describe "blocks_statistics" do
+    test "it returns the count of blocks for the latest daily periods", %{
+      conn: conn,
+      store: store
+    } do
+      st1_index = {{:blocks, :all}, :day, 1_000}
+      st2_index = {{:blocks, :all}, :day, 2_000}
+      st3_index = {{:blocks, :all}, :day, 3_000}
```

Comment:
may we use days that are easier to calculate? 1 to 355, 356 and some close to 356 * 2